packages/common/src/resources/string-to-blob.ts

```diff
@@ -1,17 +1,21 @@
 /**
  * Given a string, return it as a blob
+ * NOTE: USE objectToJsonBlob if you're saving a JSON resource!!!
  * NOTE: This is not currently supported in Node
  * @param {string} the string
  */
-export function stringToBlob(s: string): any {
+export function stringToBlob(
+  s: string,
+  options: BlobPropertyBag = { type: "application/octet-stream" }
```

Comment:
how about we make this param `type` and we default it to `application/octet-stream`? the only other thing the Blob constructor takes in it's options is an `endings`

Reply:
Sure.